internal/pkg/utils/testproject/snapshot.go

```diff
@@ -175,9 +175,20 @@ func (p *Project) NewSnapshot() (*fixtures.ProjectSnapshot, error) {
 	var tables []*keboola.Table
 	grp.Go(func() error {
 		req := p.keboolaProjectAPI.
-			ListTablesRequest(p.defaultBranch.ID, keboola.WithBuckets(), keboola.WithColumns()).
-			WithOnSuccess(func(_ context.Context, apiTables *[]*keboola.Table) error {
-				tables = append(tables, *apiTables...)
+			ListTablesRequest(p.defaultBranch.ID).
+			WithOnSuccess(func(ctx context.Context, apiTables *[]*keboola.Table) error {
```

Comment:
Please add a comment for next generations:
- `The table list does not contain the "definition" field, so we have to load the tables separately.`